src/ddl/v4_schema.sql

```diff
@@ -0,0 +1,212 @@
+--       and resetting it will ultimately cause PK collisions.
+--       To restore the counter, a row must be written with a `signal_data_id` value greater than the maximum
+--       of its values in the other tables.
+CREATE TABLE signal_load (
+    `signal_data_id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `signal_key_id` BIGINT(20) UNSIGNED,
+    `geo_key_id` BIGINT(20) UNSIGNED,
+    `demog_key_id` BIGINT(20) UNSIGNED,  -- TODO: for future use ; also rename s/demog/stratification/
+    `issue` INT(11),
+    `data_as_of_dt` DATETIME(0),  -- TODO: for future use ; also "as_of" is problematic and should be renamed
+    `source` VARCHAR(32) NOT NULL,
+    `signal` VARCHAR(64) NOT NULL,
+    `geo_type` VARCHAR(12) NOT NULL,
+    `geo_value` VARCHAR(12) NOT NULL,
+    `time_type` VARCHAR(12) NOT NULL,
+    `time_value` INT(11) NOT NULL,
+    `reference_dt` DATETIME(0),  -- TODO: for future use
+    `value` DOUBLE NULL DEFAULT NULL,
+    `stderr` DOUBLE NULL DEFAULT NULL,
+    `sample_size` DOUBLE NULL DEFAULT NULL,
+    `lag` INT(11) NOT NULL,
+    `value_updated_timestamp` INT(11) NOT NULL,
+    `computation_as_of_dt` DATETIME(0),  -- TODO: for future use ; also "as_of" is problematic and should be renamed
+    `is_latest_issue` BINARY(1) NOT NULL DEFAULT '0',
+    `missing_value` INT(1) NULL DEFAULT '0',
+    `missing_stderr` INT(1) NULL DEFAULT '0',
+    `missing_sample_size` INT(1) NULL DEFAULT '0',
+    `legacy_id` BIGINT(20) UNSIGNED, -- not used beyond import of previous data into the v4 schema
+    `compressed_signal_key` VARCHAR(100),
+    `compressed_geo_key` VARCHAR(100),
+    `compressed_demog_key` VARCHAR(100),  -- TODO: for future use ; also rename s/demog/stratification/
+    `process_status` VARCHAR(2) DEFAULT 'l', -- using codes: 'i' (I) for "inserting", 'l' (L) for "loaded", and 'b' for "batching"
+        -- TODO: change `process_status` default to 'i' (I) "inserting" or even 'x'/'u' "undefined" ?
+
+    PRIMARY KEY (`signal_data_id`) USING BTREE,
+    INDEX `comp_signal_key` (`compressed_signal_key`) USING BTREE,
+    INDEX `comp_geo_key` (`compressed_geo_key`) USING BTREE
+) ENGINE=InnoDB AUTO_INCREMENT=4000000001;
+
+
+CREATE OR REPLACE VIEW signal_history_v AS
+    SELECT
+        0 AS is_latest_issue, -- provides column-compatibility to match `covidcast` table
+        -- ^ this value is essentially undefined in this view, the notion of a 'latest' issue is not encoded here and must be drawn from the 'latest' table or view or otherwise computed...
+        NULL AS direction, -- provides column-compatibility to match `covidcast` table
+        `t2`.`source` AS `source`,
+        `t2`.`signal` AS `signal`,
+        `t3`.`geo_type` AS `geo_type`,
+        `t3`.`geo_value` AS `geo_value`,
+        `t1`.`signal_data_id` AS `signal_data_id`, -- TODO: unnecessary  ...remove?
+        `t1`.`demog_key_id` AS `demog_key_id`, -- TODO: for future use ; also rename s/demog/stratification/  ...remove?
```

Comment:
there's a bunch of these TODOs everywhere, maybe they're documented somewhere that I haven't followed, but do we have a path to resoving?

Reply:
there are many TODOs peppered throughout this changeset...  as mentioned in the initial PR message, "look for TODOs in the code just added and evaluate priorities thereof"

Reply:
@krivard write up dt terminology as an issue